pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -1334,7 +1417,131 @@ def bam_aggregation(bam_files):
                         "gnos_repo": set([bam['gnos_repo']])
                 }
 
-    return aggregated_bam_info
+    aggregated_bam_info_new['WGS'] = aggregated_bam_info
```

Comment:
The code below is a bit too repetitive, we should simplify it. But let's do it later, we need to get it work first.